presto-main/src/main/java/com/facebook/presto/memory/MemoryResource.java

```diff
@@ -48,4 +50,12 @@ public MemoryInfo getMemoryInfo(MemoryPoolAssignmentsRequest request)
         taskManager.updateMemoryPoolAssignments(request);
         return memoryManager.getInfo();
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public MemoryInfo getMemoryInfo()
+    {
+        taskManager.updateMemoryPoolAssignments(new MemoryPoolAssignmentsRequest("", 0, new ArrayList<>()));
```

Comment:
It's fine to just add a test for GET. Adding a test for POST (which is pre-existing functionality) should be a different change, outside the scope of this PR.